tests/ops_stablehlo_prettyprint.mlir

```diff
@@ -0,0 +1,229 @@
+// RUN: stablehlo-opt -split-input-file %s | FileCheck %s
+// RUN: stablehlo-opt -split-input-file %s | stablehlo-opt | FileCheck %s
+
+// -----
```

Comment:
Typically, `-split-input-file` and `// -----` are used in cases when we need to check for parse errors (because a single parse error invalidates the whole file). If parse errors are not involved, we can drop both the flag and the separator.

Reply:
updated - removed flag and deleted separators